packages/fl-exp/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "fl-exp",
```

Comment:
Should we rename to `@functionless/fl-exp` to test package deployment? Forget if we transferred ownership of `fl-exp`, think that is still on my account.

Reply:
Will do - that will also test the deployment of a package that does not exist. I've seen that fail some times so good to check.